src/Tizen.NUI/src/public/Window.cs

```diff
@@ -1310,5 +1310,52 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         {
             Interop.Window.delete_Window(swigCPtr);
         }
+
+        /// <summary>
+        /// FrameCallbackType
+        /// </summary>
+        /// <param name="frameId"></param>
```

Comment:
frameId The Id to specify the frame. It will be passed when the callback is called.